tools/integration_tests/implicitdir/implicitdir_test.go

```diff
@@ -130,9 +155,30 @@ func TestMain(m *testing.M) {
 	}
 
 	log.Printf("Test log: %s\n", logFile)
+
+	// tmp dir setup
+	tmpDir, err := ioutil.TempDir(mntDir, "tmpDir")
```

Comment:
Same here. Please avoid any ioutil methods in new code.

Reply:
Done